tests/cursor/jsfmt.spec.js

```diff
@@ -6,3 +6,11 @@ test("translates cursor correctly in basic case", () => {
     cursorOffset: 1
   });
 });
+
+test("keeps cursor inside formatted node", () => {
+  const code = "return         15";
+  expect(prettier.formatWithCursor(code, { cursorOffset: 15 })).toEqual({
+    formatted: "return 15;\n",
+    cursorOffset: 8
+  });
```

Comment:
Would be nice to write a small wrapper where you can add `{{cursor}}` inside of the string. It would greatly help review those changes.